### ISSUE TITLE
Metrolist link component

### ISSUE THREAD

**Issue by AmyWipfler** [CityOfBoston/boston.gov-d8#1771]
To connect with Amy's workflow and ID the current project page on Metrolist via Drupal

**Comment by lizcormack-bos** [comment]
Here's the final component (also in Specs)!

![Screen Shot 2021-01-07 at 11.55.28 AM.png](https://images.zenhubusercontent.com/5f89dbb0e9099f29a126af1d/e36bc6ed-54c6-408a-b7f2-de2169d59934)

Should link to the Metrolist Search listings here: https://www.boston.gov/metrolist/search

**Comment by lizcormack-bos** [comment]
@AmyWipfler I can't remember what the verdict was for what would trigger this opponent. Do we have a plan on record for that, or should we discuss?

**Comment by AmyWipfler** [comment]
Yep! It's above in the description. We will have this trigger whenever a Project is associated with a Development (the object that ML uses). For now manual, but doable - could be a workflow in future

**Comment by rachelsater** [comment]
#1681

**Comment by rachelsater** [comment]
@carolinestj Have you seen the Metrolist creative in XD? DND wants to add it to their site but I couldn't find it anywhere!
![image](https://user-images.githubusercontent.com/72876352/146441816-1ee45823-d257-4e35-a3fe-e88e0a873a8d.png)

**Comment by carolinestj** [comment]
See if you can access this! https://drive.google.com/file/d/1KOI1aWePz2iL3j2NDaXspZ-RYoRHEEjw/view

**Comment by rsater-storycorps** [comment]
@carolinestj wow! thank you! I didn't see the XD mockup file though. Is it there and I missed it?

**Comment by carolinestj** [comment]
@rachelsater yes that whole link is the XD files so just download it and it should all be there!

**Comment by rachelsater** [comment]
@carolinestj thank you!!! @padora-cob the mock up is ready for you!

**Comment by rachelsater** [comment]
@carolinestj They want to change the copy from “New units available. Explore housing opportunities on Metrolist.” to “ Explore affordable housing opportunities on Metrolist.” Is that something you update or the developer makes that change?

**Comment by carolinestj** [comment]
@rachelsater I believe a developer would make that change since it's just copy, if they don't have edit access themselves

**Comment by rachelsater** [comment]
@carolinestj great thanks!
